### PR TITLE
docs: Update README.md and CREATE.md with more info for Lazy users

### PR DESCRIPTION
**Pull request by jbrilha**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,6 +50,17 @@ Install via any package management system, for example, paq:
...
 ```
 
+Via Lazy:
+
+```lua
+return {
+	"rktjmp/lush.nvim",
+
+    -- if you wish to actually use your own colorscheme:
+    -- { dir = '/absolute/path/to/colorscheme', lazy = true },
```
Comment (rktjmp) [inline README.md:60]: - Fix indentation to use 4 spaces instead of tab
- Remove blank line
- Remove "actually" word

File: CREATE.md [hunk 2]
```diff
@@ -142,6 +142,19 @@ use '~/projects/cool_name'
...
+
+-- alternatively, if you organize your plugins by having a seperate file for each
+-- one in the dedicated plugins directory, you should instead call setup in 
+-- init.lua as you would nornally:
+require("lazy").setup("plugins")
+
+-- and treat Lush and your colorscheme directory as you would any other plugin,
+-- for example in lush.lua:
+return {
+	"rktjmp/lush.nvim",
```
Comment (rktjmp) [inline CREATE.md:154]: 4 spaces indentation

File: CREATE.md [hunk 3]
```diff
@@ -142,6 +142,19 @@ use '~/projects/cool_name'
 require("lazy").setup({
     { dir = '/home/your_username_here/projects/cool_name', lazy = true },
 }
+
```
Comment (rktjmp) [inline CREATE.md:145]: Can you please split this into a separate code block for each file, so its more obvious they're different files, and include the example filename/path.

Actually lets just pull all the wordy bits out as regular text too (I know this is a bit different to what I said before in your previous issue :pray: ), eg

Alternatively, if you organize your plugins by having a seperate file for each...
```lua
-- ~/.config/nvim/init.lua
<init.lua content>
```

And treat Lush and your colorscheme...

```lua
-- ~/.config/nvim/lua/plugins/lush.lua (or whatever it should be)
<plugin contents>
```

File: CREATE.md [hunk 4]
```diff
@@ -142,6 +142,19 @@ use '~/projects/cool_name'
...
+-- alternatively, if you organize your plugins by having a seperate file for each
```
Comment (rktjmp) [inline CREATE.md:146]: `:s/seperate/separate`

File: CREATE.md [hunk 5]
```diff
@@ -142,6 +142,19 @@ use '~/projects/cool_name'
...
+-- one in the dedicated plugins directory, you should instead call setup in
```
Comment (rktjmp) [inline CREATE.md:147]: `:s/in the/in a/`

**Review by rktjmp — CHANGES_REQUESTED** [review]
Thanks, see review

**Review by jbrilha — COMMENTED** [review]
Applied all changes from review, let me know if there's anything else

**Comment by rktjmp** [comment]
Looks good, thanks!

**Pull request merged**